civihr_employee_portal/views/views_export/views_users_list.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+$view = new view();
```

Comment:
This sounds familiar, but I forget why we have the same code in two places. Can you explain?

Reply:
tbh, I'm not sure why, I only noticed that all the views have a copy in the views_export folder and I did that to follow the convention.

Reply:
I guess it's auto-exported so I'm not gonna fight with Drupal over this